projects/resettlement-passport-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/entity/StaffTeam.kt

```diff
@@ -0,0 +1,48 @@
+package uk.gov.justice.digital.hmpps.entity
+
+import jakarta.persistence.*
+import org.springframework.data.annotation.CreatedBy
+import org.springframework.data.annotation.CreatedDate
+import org.springframework.data.annotation.LastModifiedBy
+import org.springframework.data.annotation.LastModifiedDate
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import java.io.Serializable
+import java.time.ZonedDateTime
+
+@EntityListeners(AuditingEntityListener::class)
+@Entity
+@Table(name = "staff_team")
+@IdClass(StaffTeamId::class)
+class StaffTeam(
```

Comment:
Do you still need to define the Staff and StaffTeam entities in this project?

If they're still used for the appointment endpoint, I guess we can make them `@Immutable` here and remove the created/lastUpdated columns now?

Reply:
yeah i did need to define them as i wanted to assert they have been created properly with all the fields in the tests